lib/y2lan_restart_common.pm

```diff
@@ -64,8 +65,8 @@ sub initialize_y2lan
         assert_script_run("systemctl show -p ActiveState firewalld.service | grep ActiveState=inactive");
     }
     # enable debug for detailed messages and easier detection of restart
-    assert_script_run 'sed -i \'s/DEBUG="no"/DEBUG="yes"/\' /etc/sysconfig/network/config';
-    type_string "journalctl -f|egrep -i --line-buffered 'shutting down|ifdown all' > journal.log &\n";
+    assert_script_run 'sed -i -E \'s/(WICKED_LOG_LEVEL).*/\1="info"/\' /etc/sysconfig/network/config';
+    type_string "journalctl -f|egrep -i --line-buffered 'shutting down|ifdown all|Stopping wicked' > journal.log &\n";
     assert_script_run '> journal.log';    # clear journal.log
```

Comment:
I am wandering if this function is used or might be used on tests with Network Manager. Should I add a check that would determine which manager is used and what short of configuration for getting debug messages should be added. Also, id 'shutting down|ifdown all' should be removed from egrep command.